commands/management/commands/run_coverage.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
+import os
+import sys
+
+
+class Command(BaseCommand):
+    help = 'Runs coverage'
+
+    def handle(self, *args, **kwargs):
+        if os.getenv('PY_ENV') != 'test':
+            sys.exit('Please ensure PY_ENV environment variable is set to \'test\'')
+        os.system('pipenv run ./manage.py test -v 2 --noinput\
```

Comment:
we need `call_command` here.

Reply:
Can't do that. Please refer to this [issue](https://github.com/jazzband/django-nose/issues/256) in `django-nose`

Reply:
Interesting.